app/Models/Article.php

```diff
@@ -23,9 +23,10 @@ class Article extends Model
         'content',
         'category_id',
         'user_id',
+        'author_id',
         'approved',
-        // 'articleable_type',
-        // 'articleable_id',
+        'articleable_type',
```

Comment:
Hello Susana, there is no need to add type and id in fillable, fillable means the data that will be added manually, like slug is added automatically there is no need to add it into fillable, so as to morph relation is handled automatically , see the following.

![image](https://github.com/eduCyber-HRmagazine/adminDashboard-backend/assets/96319423/2b5dba05-f7c5-48ec-9df7-f2ba5e2f5d88)